.babelrc

```diff
@@ -1,6 +1,3 @@
 {
-    "presets": [
-        "@babel/preset-env",
-        "@babel/preset-react"
-    ]
+  "presets": ["@babel/preset-env", "@babel/preset-react"]
```

Comment:
@walimike , You know you this marks makes readability hard. It was readable the way it was before you changed it.